### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by krate27**
Please review:

Added cart functionality

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,16 +5,18 @@ import { Body } from "./Routes/homeBody/body.jsx";
...
 import { NavBar } from "./Components/Navbar/navbar.jsx";
+import { CartCard } from "./Components/CartCard/cartCard.jsx";
 function App() {
   return (
     <div className="App">
        <NavBar />
       <Routes>
         <Route path="/" element={<Body />} />
         <Route path="/shopNow" element={<ShopNow />} />
+        <Route path="/cart" element={<CartCard />} />
```
Comment (shivampandey0) [inline src/App.js:16]: Shouldn't it go to the `Cart` route? instead of `CartCard`.
maybe a naming issue.

File: src/App.js [hunk 2]
```diff
@@ -5,16 +5,18 @@ import { Body } from "./Routes/homeBody/body.jsx";
 import { ShopNow } from "./Routes/Products/shopNow.jsx";
 import { Footer } from "./Components/Footer/footer.jsx";
 import { NavBar } from "./Components/Navbar/navbar.jsx";
+import { CartCard } from "./Components/CartCard/cartCard.jsx";
 function App() {
   return (
     <div className="App">
        <NavBar />
       <Routes>
         <Route path="/" element={<Body />} />
```
Comment (shivampandey0) [inline src/App.js:14]: It should got to `Home` or `Homepage` not `Body`

File: src/Components/Filters/filter.jsx [hunk 3]
```diff
@@ -19,9 +21,9 @@ const Filter = () => {
       <div className="filter-wrapper">
         <div className="filter-class">
           <h3>FILTERS</h3>
-          <a href="" className="create-acc">
+          <button className="create-acc underline" onClick={()=>dispatch({type:Abbreviations.CLEARFILTER})} style={{border:'none', backgroundColor:'#e0e0eb', cursor:'pointer', fontWeight:'bold'}}>
```
Comment (shivampandey0) [inline src/Components/Filters/filter.jsx:24]: inline CSS changes shouldn't be done.

File: src/Components/Filters/filter.jsx [hunk 4]
```diff
@@ -167,36 +277,78 @@ const Filter = () => {
...
+                    dispatch({ type: Abbreviations.DEADPOOL });
+                  }}
+                />{" "}
+                Deadpool
               </label>
             </li>
             <li>
-              <label for="filter-label" className="label-selector">
-                <input type="checkbox" className="checkbox-select" /> Loki
+              <label htmlFor="loki" className="label-selector">
```
Comment (shivampandey0) [inline src/Components/Filters/filter.jsx:340]: This file became too big. Make components for reused elements and use them to keep the code clean.

File: src/Components/ProductListing/productListing.jsx [hunk 5]
```diff
@@ -1,39 +1,37 @@
...
-      <div className="all-card-collection" style={{margin:'0rem'}}>
+      <div className="all-card-collection" style={{ margin: "0rem" }}>
         <div className="card-container">
           <picture className="background-img">
             <img
               src={img}
               alt="image"
               className="basic-image"
-              style={{width:'20rem'}}
+              style={{ width: "20rem" }}
```
Comment (shivampandey0) [inline src/Components/ProductListing/productListing.jsx:25]: inline CSS shouldn't be used

File: src/Components/ProductListing/productListing.jsx [hunk 6]
```diff
@@ -1,39 +1,37 @@
...
+  } = products;
   return (
     <>
-      <div className="all-card-collection" style={{margin:'0rem'}}>
+      <div className="all-card-collection" style={{ margin: "0rem" }}>
         <div className="card-container">
           <picture className="background-img">
             <img
               src={img}
               alt="image"
```
Comment (shivampandey0) [inline src/Components/ProductListing/productListing.jsx:23]: `alt` should be a meaningful string instead of just `image`

**Review by shivampandey0 — APPROVED** [review]
Couldn't check everything because of too long PR. 
- Added some suggestions

Can merge now but fix these issues in future PRs

**Review by junaidshaikh-js — APPROVED** [review]
Good work! 

Clean code, can be merged.

**Pull request merged**